### DISCUSSION TITLE
Latest release

### DISCUSSION THREAD

**Discussion by suraj-tekdi** [Sunbird-Saral/Community#82] [General]
Hi Team,

I need to know, what is the next release coming for the sunbird-saral? what is the latest released branch?

**Comment by manojLondhe** [comment]
I also want to know what branch to use
- we have fixed some security issues and want to send PR for same
pls let know which branch to use

**Comment by rhwarrier** [comment]
[harish-challa](https://github.com/harish-challa) please update ?

**Comment by parthlawate** [comment]
Hello @rhwarrier you had mentioned Saral 2.0 has enabled quite a few new features. However i was not able to find it clearly on the roadmap or release notes. Could you help point in the right direction ? 

https://saral.sunbird.org/use/roadmap
https://saral.sunbird.org/engage/releases

**Comment by rsankagi** [comment]
https://github.com/Sunbird-Saral/Project-Saral/releases/tag/v1.6.1


Here is the latest release, you can take a pull from main branch.
